snorkel/labeling/model/logger.py

```diff
@@ -51,7 +51,7 @@ def log(self, metrics_dict: Dict[str, float]) -> None:
         >>> logger = Logger(log_train_every=5)
         >>> metrics_dict = {"train/loss": 5.00}
         >>> logger.log(metrics_dict)
-        [0 epochs]: TRAIN:[loss=5.000]
+        [-1 epochs]: TRAIN:[loss=5.000]
```

Comment:
Also weird...accepting to unblock but let's ask ourselves if this even needs an example in the docstring